src/napari_sphinx_theme/theme/napari_sphinx_theme/layout.html

```diff
@@ -100,6 +100,15 @@
     </div>
 
     <script src="{{ pathto('_static/scripts/napari-sphinx-theme.js', 1) }}"></script>
+    <script>
```

Comment:
Should this JS be in the main js file listed above instead? napari-sphinx-theme.js

Reply:
Sure, maybe, but this seem to be a file that is built in some way, but I have no clue how.
I've tried to follow some of the docs but it often refer to non-existing files/folder. 
Like all of https://napari.org/napari-sphinx-theme/contribute/index.html talk about "pydata-sphix-theme", and https://github.com/napari/napari-sphinx-theme/blob/main/CONTRIBUTING.md link to https://github.com/napari/napari-sphinx-theme/blob/main/docs/contributing.md which is non-existant.

At least with this I was able to make it work locally.

I'm happy to take the time to figure it out, but it might take me much more time than someone more familiar with how this works.